content/en/tracing/glossary/_index.md

```diff
@@ -106,6 +107,12 @@ You can use custom instrumentation by embedding tracing code directly into your
 
 To learn more, read [Application Instrumentation][26].
 
+## Baggage
+
+Baggage allows you to propagate key-value pairs (also known as baggage items) across service boundaries in a distributed system. Unlike trace context, which focuses on trace identifiers, baggage allows for the transmission of business data and other contextual information alongside traces.  
+
+To learn more, read supported [propagation formats][27] for your application's language.
```

Comment:
```suggestion
To learn more, read about [supported propagation formats][27] for your application's language.
```

Reply:
I'm not able to make a suggestion on reference link `[27]` (line 145), but suggesting to update the link for the specific section that's being mentioned:
`/tracing/trace_collection/trace_context_propagation/#supported-formats`

Reply:
Addressed!